gl unit test failure: check activate and deactivating fill and stroke

<!--
Hi there! If you are here to report a bug, or to discuss a feature (new or existing), you can use the below template to get started quickly. Fill out all those parts which you're comfortable with, and delete the remaining ones.

To check any option, replace the "[ ]" with a "[x]". Be sure to check out how it looks in the Preview tab!

Feel free to remove any portion of the template that is not relevant for your issue.
-->

#### Nature of issue?

- [x] Found a bug

#### Most appropriate sub-area of p5.js?

- [x] WebGL

One of the WebGL unit tests doesn't pass. Here's what it says:

```
AssertionError: fill shader still active after noFill(): expected false to be true
    at Function.assert.isTrue (http://localhost:8000/node_modules/chai/chai.js:2483:31)
    at Context.<anonymous> (unit/webgl/stroke.js:50:14)
```

--- a/test/unit/webgl/p5.Shader.js
+++ b/test/unit/webgl/p5.Shader.js
@@ -128,12 +128,11 @@ suite('p5.Shader', function() {
       );
     });
     test('Color Shader definition', function() {
-      var expectedAttributes = ['aPosition', 'aNormal', 'aTexCoord'];
+      var expectedAttributes = ['aPosition'];
 
       var expectedUniforms = [
         'uModelViewMatrix',
         'uProjectionMatrix',
-        'uNormalMatrix',
         'uMaterialColor'
       ];
 
@@ -162,7 +161,7 @@ suite('p5.Shader', function() {
       );
     });
     test('Normal Shader definition', function() {
-      var expectedAttributes = ['aPosition', 'aNormal', 'aTexCoord'];
+      var expectedAttributes = ['aPosition', 'aNormal'];
 
       var expectedUniforms = [
         'uModelViewMatrix',

--- a/test/unit/webgl/stroke.js
+++ b/test/unit/webgl/stroke.js
@@ -48,7 +48,7 @@ suite('stroke WebGL', function() {
         'stroke shader not active after stroke()'
       );
       assert.isTrue(
-        myp5._renderer._doFill,
+        !myp5._renderer._doFill,
         'fill shader still active after noFill()'
       );
       done();

